bucket/oculante.json

```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://github.com/woelper/oculante",
+    "description": "A minimalistic crossplatform image viewer written in Rust",
+    "bin": "oculante.exe",
+    "version": "0.6.32",
+    "url": "https://github.com/woelper/oculante/releases/download/0.6.32/oculante.exe",
+    "hash": "b68534ff12f301044459476b9c3f7a3ccc8301083f5eb0594e2bfee2f9907b42",
+    "license": "MIT",
```

Comment:
Hi @eugenesvk, it doesn't seem a serious issue but can you please fix the order of the fields in manifest as described in the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md)?

Reply:
Done
Interesting why doesn't the auto-update tool fix the order, it did some other reformatting of the file?

Reply:
:+1:

...I guess the check only validates against [schema](https://raw.githubusercontent.com/ScoopInstaller/Scoop/master/schema.json) presence and values of fields but not the order.

Reply:
yeah, added a suggestion to make it also enforce the order https://github.com/ScoopInstaller/Scoop/issues/5292